core/src/main/java/com/scalar/db/transaction/common/AbstractDistributedTransactionManager.java

```diff
@@ -111,13 +113,18 @@ public DistributedTransaction resume(String txId) throws TransactionException {
                         + "It might have been expired"));
   }
 
-  @VisibleForTesting
-  public class ActiveTransaction extends AbstractDistributedTransaction {
+  protected DistributedTransaction wrap(DistributedTransaction transaction)
```

Comment:
Personally, `wrap` doesn't sound very meaningful, so how about renaming it to `activate` or something?
```suggestion
  protected DistributedTransaction activate(DistributedTransaction transaction)
```

Reply:
`withActiveStatus` also sounds okay? It might be a bit verbose.

Reply:
This method wraps the transaction object with `ActiveTransaction` and `TransactionStateManagement` for now (we might want to add another wrapper in the future). I feel the names `activate`, `withActiveStatus` and `asActive` focus on only `ActiveTransaction`. Is that correct?

Reply:
I thought it's related to TransactionStateManagement as well because the initial value set in the constructor is ACTIVE.

Reply:
Okay, so I will rename it to `activate`. Thanks.

Reply:
Renamed it in f97d5eeb724d40d80f7ad21812bf10757ae00a0e. Thanks.